app/learning/index.md

```diff
@@ -5,204 +5,98 @@ title: Getting started with Yeoman
 sidebar: sidebars/learning.md
 ---
 
-The Yeoman workflow is comprised of three core tools for improving your productivity and satisfaction when building a web app. These tools are:
+Yeoman is a generic scaffolding system allowing you to create any kind of app. It'll allow you to rapidly get started on new projects and streamline the maintenance of existing projects.
 
-* [yo](https://github.com/yeoman/yo) - the scaffolding tool from Yeoman
-* A package manager, such as [bower](http://bower.io/) or [npm](https://www.npmjs.com/).
-* A build tool, such as [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/).
+Yeoman is language agnostic. It can generates projects in any languages (Web, Java, Python, C#, etc.)
 
-Each of these projects are independently maintained by their respective communities, but work well together as a part of a prescriptive workflow for keeping you effective. Let’s walk through what these binaries can do.
+Yeoman by itself doesn't make any decision. Yeoman runs _generators_ who're making every decision. A generator is basically a plugin in the Yeoman environment. There's more than [1500 publicly available generators](/generators) and you can [create your own](/authoring) if public ones are not matching your team workflow; Yeoman is always the right choice for your scaffolding needs.
```

Comment:
`Yeoman by itself doesn't make any decisions. Every decision is made by _generators_ which are basically plugins in the Yeoman environment. There's a _[lot of publicly available generators](/generators)_ and its easy to [create a new one](/authoring) to match any workflow. Yeoman is always the right choice for your scaffolding needs.`